index.php

```diff
@@ -23,6 +23,10 @@
     ini_set('display_startup_errors', 1);
     ini_set('display_errors', 1);
     error_reporting(-1);
```

Comment:
Since PHP 8.0.0, error_reporting is set to E_ALL by default so this can be removed